src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
```

Comment:
Author tag is not recommended as it can easily get outdated and does not provide much value. (Severity: low)